orchestrator/modules/tools/discovery/handlers_documents.py

```
@@ -302,3 +302,201 @@ async def upload_document(db: Session, workspace_id: UUID, params: Dict[str, Any
+
+
+def _resolve_agent_team(db: Session, agent_id: Any) -> Optional[str]:
+    """Look up an agent's team for retrieval scoping. None when unknown."""
+    if not agent_id:
+        return None
+    try:
+        from core.models import Agent
+
+        row = db.query(Agent).filter(Agent.id == int(agent_id)).first()
+        return getattr(row, "team", None) if row else None
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)